priv/templates/extended_bin

```diff
@@ -674,8 +690,17 @@ EPMD_MODULE="$(grep '^-epmd_module' "$VMARGS_PATH" || true)"
 if [ "$EPMD_MODULE" ]; then
     DIST_ARGS="${DIST_ARGS} ${EPMD_MODULE}"
 fi
+INET_DIST_USE_INTERFACE="$(grep '^-kernel  *inet_dist_use_interface' "$VMARGS_PATH" || true)"
+if [ "$INET_DIST_USE_INTERFACE" ]; then
+    DIST_ARGS="${DIST_ARGS} ${INET_DIST_USE_INTERFACE}"
+fi
 
 if [ "$ERL_DIST_PORT" ]; then
+    if [ "$INET_DIST_USE_INTERFACE" ]; then
+        ADDRESS="$(addr_tuple_to_str "${INET_DIST_USE_INTERFACE#*inet_dist_use_interface }"):$ERL_DIST_PORT"
```

Comment:
Not sure what is going on here with `${INET_DIST_USE_INTERFACE#*inet_dist_use_interface }`?

And have to make sure it is posix shell compliant? Not bash.

Reply:
Oh, looks like shellcheck caught something else so this must have passed and thus work ok with posix shell.

But I still don't know what it is doing :)

Reply:
> `${INET_DIST_USE_INTERFACE#*inet_dist_use_interface }`

At that point `INET_DIST_USE_INTERFACE` is e.g. `-kernel inet_dist_use_interface {127,0,0,1}`.  The `#*inet_dist_use_interface ` thing is stripping everything up to the actual address tuple.

> And have to make sure it is posix shell compliant?

It is.

Reply:
Shall I try to come up with something less magic, more readable?

Reply:
Ahh, no, this is good and makes sense. I was misreading (still early :) and thought it was like `${ADDRESS#*inet_dist_use_interface }` so had no idea why `inet_dist_use_interface` was involved when I thought it was already just the address :)